ecosystem/typescript/sdk/examples/typescript/http2-fetch-h2/index.ts

```diff
@@ -0,0 +1,253 @@
+    accounts.push(new AptosAccount());
+    recipients.push(new AptosAccount());
+  }
+  console.log("accounts created");
+  console.log(timer.time());
+
+  // funds accounts
+  const funds: Promise<any>[] = [];
+
+  for (let i = 0; i < accounts.length; i++) {
+    funds.push(faucet(accounts[i], 100000000));
+  }
+  for (let i = 0; i < recipients.length; i++) {
+    funds.push(faucet(recipients[i], 0));
+  }
+  await Promise.all(funds);
+  // sleeps to let faucet do its work without the need to implement
+  // waitForTransaction in this new client
+  await sleep(15000); // 15 seconds
+  console.log("accounts funded");
+  console.log(timer.time());
+
+  // read accounts
+  const balances: Promise<any>[] = [];
+  for (let j = 0; j < readAmplification; j++) {
+    for (let i = 0; i < accounts.length; i++) {
+      balances.push(get(`accounts/${accounts[i].address().hex()}`));
+    }
+  }
+
+  await Promise.all(balances);
+  console.log("accounts checked");
+  console.log(timer.time());
+
+  // initialize accounts with sequence number
+
+  // array to hold the sequence number class initialization of an account
+  const accountSequenceNumbers: AccountSequenceNumbers[] = [];
+  // array to hold prmoises to fetch account current sequence number
+  const awaitSequenceNumbers: Promise<void>[] = [];
+  for (let i = 0; i < accounts.length; i++) {
+    accountSequenceNumber = new AccountSequenceNumbers(accounts[i]);
+    awaitSequenceNumbers.push(accountSequenceNumber.initialize());
+    accountSequenceNumbers.push(accountSequenceNumber);
+  }
+
+  await Promise.all(awaitSequenceNumbers);
+  console.log("accounts initialized");
+  console.log(timer.time());
+
+  // submit transactions
+  let transactionsHashes: string[] = [];
+  for (let i = 0; i < firstPass; i++) {
+    for (let j = 0; j < accountsCount; j++) {
+      let sender = accounts[j];
+      let recipient = recipients[j].address().hex();
+      let sequenceNumber: bigint = await accountSequenceNumbers[j].nextSequenceNumber();
+      let txnHash = await transafer(sender, recipient, sequenceNumber, 1);
+      transactionsHashes.push(txnHash);
+    }
+  }
+
+  transactionsHashes = await Promise.all(transactionsHashes);
+  console.log("transactions submitted");
+  console.log(timer.time());
+  // check for transactions
+  const waitFor: Promise<void>[] = [];
+  for (let i = 0; i < transactionsHashes.length; i++) {
+    waitFor.push(accountSequenceNumber!.synchronize());
+  }
+
+  await Promise.all(waitFor);
+  console.log("transactions commited");
+  console.log(timer.time());
+
+  exit(0);
+}
+
+async function transafer(sender: AptosAccount, recipient: string, sequenceNumber: bigint, amount: number) {
+  const token = new TxnBuilderTypes.TypeTagStruct(TxnBuilderTypes.StructTag.fromString("0x1::aptos_coin::AptosCoin"));
+
+  const entryFunctionPayload = new TxnBuilderTypes.TransactionPayloadEntryFunction(
+    TxnBuilderTypes.EntryFunction.natural(
+      "0x1::coin",
+      "transfer",
```

Comment:
fyi, for future ease of use: `0x1::aptos_account::transfer_coins` will create the account / register the coin if it doesn't exist